test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -375,8 +375,6 @@ private static void VerifyFile(string file, string content)
                 case ".log":
                     var expected = content.Split(new char[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries).OrderBy(_ => _).ToArray();
                     var actual = File.ReadAllLines(file).OrderBy(_ => _).ToArray();
-                    // TODO: Configure github token in CI to get rid of github rate limit,
-                    // then we could remove the wildcard match
                     if (expected.Any(str => str.Contains("*")))
```

Comment:
the wildcard match is not removed?

Reply:
no, there are other use cases for wildcard match, e.g., some error message contains system level message info, like download failed.

Reply:
ok